src/scales/scale.time.js

```diff
@@ -71,7 +71,7 @@ function arrayUnique(items) {
 
 	for (i = 0, ilen = items.length; i < ilen; ++i) {
 		item = items[i];
-		if (!hash[item]) {
+		if (item !== null && !hash[item]) {
```

Comment:
We can filter null out when the `timestamps` array is built, but not for the `labels` array because it is copied to `datasets` later.

May be we can do this way.
```
labels = arrayUnique(labels).sort(sorter);
if (labels[0] === null) {
    labels.shift();
}
```
```
timestamps = arrayUnique(timestamps).sort(sorter);
if (timestamps[0] === null) {
    timestamps.shift();
}
```